.gitignore

```diff
@@ -3,7 +3,7 @@ node_modules
 !.env.example
 coverage.out
 build
-fathom.db
+*.db
```

Comment:
Any reason for this change? Are there gonna be multiple databases in the future or something?